lib/auth/auth.go

```diff
@@ -1941,6 +1941,10 @@ func (a *Server) ExtendWebSession(ctx context.Context, req WebSessionReq, identi
 	// Keep preserving the login time.
 	sess.SetLoginTime(prevSession.GetLoginTime())
 
+	if req.AccessRequestID != "" {
```

Comment:
I don't think you really need the if here. If `req.AccessRequestID` is empty, it will just set it to empty in the session, which is already the case.